meteor/lib/Settings.ts

```diff
@@ -41,7 +43,8 @@ const DEFAULT_SETTINGS: ISettings = {
 	autoRewindLeavingSegment: false,
 	disableBlurBorder: false,
 	defaultTimeScale: 1,
-	allowGrabbingTimeline: true
+	allowGrabbingTimeline: true,
+	allowUnsyncedSegments: true
```

Comment:
I suspect that with mos resyncing segments will get 'stuck' in a loop, so it might be better to disable by default at least until there is a fix for that.

This is because mos has to resync the whole rundown, so while resyncing a segment, it will likely unsync the newly playing segment.

Reply:
Addressed by https://github.com/olzzon/tv-automation-server-core/commit/bbf6d34e5fa148c2bad34d3aa22ec79079aa25bf